team-telnyx/telnyx-client-examples
# Refactor calls controller

Moves call control actions and callbacks out from `CallsController`, which will only handle DB operations on `Calls`.

## ✋ Manual testing

1. Change the primary webhook path for your call control application in the portal to `/call-control/callbacks`
2. Run the call-center app and login via a Chrome incognito window
3. Place inbound and outbound calls. Verify that app works as expected

## 🦊 Browser testing

### Desktop

### Mobile

## 📸 Screenshots
